substrate/frame/contracts/src/chain_extension.rs

```diff
@@ -289,30 +289,12 @@ impl<'a, 'b, E: Ext> Environment<'a, 'b, E, InitState> {
 
 /// Functions to use the input arguments as integers.
 impl<'a, 'b, E: Ext, S: PrimIn> Environment<'a, 'b, E, S> {
-	/// The `input_ptr` argument.
-	pub fn val0(&self) -> u32 {
-		self.inner.input_ptr
-	}
-
 	/// The `input_len` argument.
-	pub fn val1(&self) -> u32 {
+	pub fn input_len(&self) -> u32 {
 		self.inner.input_len
 	}
 }
 
-/// Functions to use the output arguments as integers.
-impl<'a, 'b, E: Ext, S: PrimOut> Environment<'a, 'b, E, S> {
-	/// The `output_ptr` argument.
-	pub fn val2(&self) -> u32 {
-		self.inner.output_ptr
-	}
-
-	/// The `output_len_ptr` argument.
-	pub fn val3(&self) -> u32 {
-		self.inner.output_len_ptr
-	}
-}
```

Comment:
if we compile a Rust contract we can't predict these values. I think that's why I removed it, It's been a while, I  can look into it again

Reply:
Test don't use the _ptr values (val0(), val1(), val3()) though since these are set by the rust compiler now instead of manually as we were doing with .wat contract